setup/web.sh

```diff
@@ -67,8 +67,7 @@ tools/editconf.py /etc/php/7.0/fpm/pool.d/www.conf -c ';' \
 # nginx configuration at /mailinabox-mobileconfig.
 mkdir -p /var/lib/mailinabox
 chmod a+rx /var/lib/mailinabox
-cat conf/ios-profile.xml \
-	| sed "s/PRIMARY_HOSTNAME/$PRIMARY_HOSTNAME/" \
+< conf/ios-profile.xml sed "s/PRIMARY_HOSTNAME/$PRIMARY_HOSTNAME/" \
```

Comment:
Same as 330 in system.sh - the cat is less efficient - but more readable

Reply:
I would suggest you run these two commands on your system from the mail-in-a-box repository:
`apt-get install shellcheck`
`bash -c 'shopt -s globstar; shellcheck -s bash **/*.sh'`
This will provide an explanation for all my changes in this pull request.

You can also see [here](https://travis-ci.org/tdulcet/mailinabox/builds/445985225#L1374) (lines 1374-2490).

Reply:
100% true that the cat is *technically* useless and "wastes" a few CPU cycles and a few kilobytes of memory. But if we have to choose between that waste and having the script be easier to read/understand, I always go with it being easier to read. I defer to Josh's preferences.